README.md

```diff
@@ -88,6 +89,9 @@ You must set all of these when you include this dependency in your project,
 however you can easily run this project as is and the above credentials will
 log you into slack so long as you set the SLACK_API_TOKEN env var.
 
+## Direct Messages
+If it makes sense for your bot to receive direct messages, then set the `allow_direct_messsages` to true, otherwise - don't. It may not make sense for your bot to receive direct messages if it relies on the channel to provide state.
```

Comment:
I would probably prefer to see a note about the fact that all direct messages will be processed even if they don't include the bot name (unlike channel messages).